packages/lesswrong/lib/collections/users/views.ts

```diff
@@ -175,7 +177,7 @@ Users.addView("tagCommunityMembers", function (terms: UsersViewTerms) {
     }
   }
 })
-ensureIndex(Users, {profileTagIds: 1})
+ensureIndex(Users, {profileTagIds: 1, deleted: 1, deletedContent: 1, karma: 1})
```

Comment:
Not sure how many indexes we have on `Users` but I'd double-check that it's not 64 before sending this out into the wild - recently noticed an issue where `Posts` wasn't picking up new indexes, causing a stray query to trigger a full table scan